# GetPointsInLasso.srv
geometry_msgs/Polygon points
sensor_msgs/PointCloud2 cloud
std_msgs/Float32 extrusion_depth
---
std_msgs/UInt8MultiArray lasso_points